src/app/containers/FrontPageSection/index.jsx

```diff
@@ -125,28 +158,14 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
       >
         {group.strapline.name}
       </SectionLabel>
-      {items.length > 1 ? (
-        <MarginWrapper firstSection={isFirstSection}>
-          <StoryPromoUl>
-            {items.map((item, index) => (
-              <StoryPromoLi key={item.id}>
-                <StoryPromoComponent
-                  item={item}
-                  sectionNumber={sectionNumber}
-                  storyNumber={index}
-                />
-              </StoryPromoLi>
-            ))}
-          </StoryPromoUl>
-        </MarginWrapper>
+      {group.strapline.name === 'Useful links' ? (
+        <UsefulLinksComponent items={items} script={script} service={service} />
       ) : (
-        <MarginWrapper firstSection={isFirstSection} oneItem>
-          <StoryPromoComponent
-            item={items[0]}
-            sectionNumber={sectionNumber}
-            storyNumber={0}
-          />
-        </MarginWrapper>
+        <StoryPromoRenderer
+          items={items}
+          firstSection={isFirstSection}
+          sectionNumber={sectionNumber}
+        />
```

Comment:
Is this logic tested in the tests? If not we should check that the container renders useful links when the strapline name is `Useful links`.